apps/exth/test/exth_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule ExthTest do
+  use ExUnit.Case
+  doctest Exth
+
+  test "greets the world" do
```

Comment:
How would you feel about removing this? Unless this is the implementation for `Packet.hello`?

Reply:
Yes, will be removed. `Exth` should be tested. I know most people don't love catch-alls, but we're getting close to a lot of repeated code.